### DISCUSSION TITLE
Prompt Format Llama 2

### DISCUSSION THREAD

**Discussion by schacki** [microsoft/autogen#1010] [Q&A]
I have been using autogan with LM Studio and Lllama 2 models. Unfortunately, the results are poor; e.g. the group chat manager is not able to decide on the next speaker.

So I am wondering, if it is an issue with the prompt format, after reading this post: https://replicate.com/blog/how-to-prompt-llama#how-to-format-prompts-in-production

So my general question is, do i need to adapt the prompt for Llama 2 models. And if so, how would i do that?

**Comment by rickyloynd-microsoft** [comment]
Reliable group chats usually depend on gpt-4 today.

**Comment by schacki** [comment]
That might be true, but still does not answer the question, if and how the prompt should be adapted.

**Comment by rickyloynd-microsoft** [comment]
Yes, and your question about better prompt writing and formatting is the more interesting research question. There's a fair amount of discussion about it on our [Discord channel](https://discord.gg/pAbnFJrkgZ).
